### ISSUE TITLE
Minecraft failed to start due to NetherEx

### ISSUE THREAD

**Issue by SolitaryPilot** [LogicTechCorp/NetherEx-Archive#190]
Heres the link it gave me for the issue; 

https://paste.dimdev.org/kaduniribi.mccrash

**Comment by LogicTechCorp** [comment]
Delete the pigtificate trade configs in the config folder.

**Comment by SolitaryPilot** [comment]
Tried that, I deleted the entirety of the trade folders, still having the same issue.

**Comment by LogicTechCorp** [comment]
Actually, you need to delete the biome configs. Before you do, would you mind telling me what is on the first line in each file?

**Comment by SolitaryPilot** [comment]
Tried opening with notepad but nothing is showing up in there, its totally blank. If it helps, I downloaded the mod and all through the twitch app thing. I'm not very good with computers at all and it seemed a pretty simply way to do it. I also have both a netherex file and a NetherEx_old file, if that makes any sort or difference. 

Note; The files in the NetherEx_old are showing things when opened. Theres not much code in there but all of them only have a { on the first line.
